packages/extras/src/lib/components/Environment/GroundProjectedEnv.svelte

```diff
@@ -1,11 +1,13 @@
 <script lang="ts">
-  import type { EnvironmentProperties } from '$lib/types/components'
   import { useThrelte } from '@threlte/core'
   import { onDestroy } from 'svelte'
   import type { Texture } from 'three'
   import { GroundProjectedEnv } from 'three/examples/jsm/objects/GroundProjectedEnv'
+  import type { GridProps } from '../Grid/Grid.svelte'
```

Comment:
GroundProjectedEnv is not part of the public API, it doesn't need external component types.

Reply:
Does this mean you don't want me to import props in GroundProjection sub-component? I did it to easier reason with types when developing it.

```ts
  import type { EnvironmentProps } from './Environment.svelte'

  type Props = EnvironmentProps

  export let groundProjection: Props['groundProjection']
  export let currentEnvMap: Texture
  ```